app/workers/clean_up_course_options_with_no_study_mode.rb

```diff
@@ -0,0 +1,23 @@
+class CleanUpCourseOptionsWithNoStudyMode
+  include Sidekiq::Worker
+
+  def perform
+    CourseOption.where('study_mode = \'0\'').each do |nil_co|
+      if nil_co.application_choices.any?
+        equivalent_course_option = CourseOption.where(
```

Comment:
`find_by` instead of `where`?

Reply:
I think `where` is better here, because I want all of them, plus not sure `find_by` works with sql fragments.